### PR TITLE
[AIG] added links

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary

- Added links for better description

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by ranbel — APPROVED** [review]

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 1]
```diff
@@ -9,19 +9,19 @@ import { Render } from "~/components";
...
+| Cacheable request size                                        | 25 MB per request                   |
+| [Custom metadata](/ai-gateway/configuration/custom-metadata/) | 5 entries per request               |
+| Datasets                                                      | 10 per gateway                      |
+| Gateways                                                      | 10 per account                      |
+| Gateway name length                                           | 64 characters                       |
+| Log storage rate limit                                        | 500 logs per second per gateway     |
+| Logs stored [paid plan](/ai-gateway/reference/pricing/)       | 10 million per gateway <sup>1</sup> |
+| Logs stored [free plan](/ai-gateway/reference/pricing/)       | 100,000 per account <sup>2</sup>    |
+| [Log size stored](/ai-gateway/observability/logging/)         | 10 MB per log <sup>3</sup>          |
+| [Logpush jobs](/ai-gateway/observability/logging/)            | 4 per account                       |
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: can this be to: https://developers.cloudflare.com/ai-gateway/observability/logging/logpush/

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 2]
```diff
@@ -9,19 +9,19 @@ import { Render } from "~/components";
...
+| [Logpush size limit](/ai-gateway/observability/logging/)      | 1MB per log                         |
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: same https://developers.cloudflare.com/ai-gateway/observability/logging/logpush/

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 3]
```diff
@@ -9,19 +9,19 @@ import { Render } from "~/components";
...
-| Gateway name length                                             | 64 characters                       |
-| Log storage rate limit                                          | 500 logs per second per gateway     |
-| Logs stored [paid plan](/ai-gateway/reference/pricing/)         | 10 million per gateway <sup>1</sup> |
-| Logs stored [free plan](/ai-gateway/reference/pricing/)         | 100,000 per account <sup>2</sup>    |
-| Log size stored                                                 | 10 MB per log <sup>3</sup>          |
-| Logpush jobs                                                    | 4 per account                       |
-| Logpush size limit                                              | 1MB per log 	                      |
+| Feature                                                       | Limit                               |
+| ------------------------------------------------------------- | ----------------------------------- |
+| Cacheable request size                                        | 25 MB per request                   |
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: https://developers.cloudflare.com/ai-gateway/configuration/caching/

File: src/content/docs/ai-gateway/reference/limits.mdx [hunk 4]
```diff
@@ -9,19 +9,19 @@ import { Render } from "~/components";
...
+| [Custom metadata](/ai-gateway/configuration/custom-metadata/) | 5 entries per request               |
+| Datasets                                                      | 10 per gateway                      |
```
Comment (kathayl) [inline src/content/docs/ai-gateway/reference/limits.mdx]: https://developers.cloudflare.com/ai-gateway/evaluations/set-up-evaluations/

**Review by kathayl — APPROVED** [review]
added comments. once those are added in this is approved

**Pull request merged**